recipes/fmi4cpp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "fmi4cpp" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NTNU-IHB/FMI4cpp/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 78616e9c86a23137a8d3a113fe6420207c3f9ea46442e1c75a01215eb2693bb7
+
+build:
+  number: 0
+  # See https://github.com/conda-forge/staged-recipes/pull/18411#issuecomment-1073669556
+  skip: true  # [osx] 
```

Comment:
How does getting a provisioned feedstock solve this error?

Reply:
This is what we did in a similar occasion in https://github.com/conda-forge/staged-recipes/pull/13677/files#r562530228, I guess `staged-recipes` does not have the logic to handle enabling newer macOS SDKs. Feel free to suggest any alternative solution if you are aware of them, thanks!

Reply:
OK.